src/core/thread/mle_router.cpp

```diff
@@ -739,9 +741,19 @@ ThreadError MleRouter::SendLinkAccept(const Ip6::MessageInfo &aMessageInfo, Neig
         aNeighbor->mState = Neighbor::kStateLinkRequest;
     }
 
-    SuccessOrExit(error = SendMessage(*message, aMessageInfo.GetPeerAddr()));
+    if (aMessageInfo.GetSockAddr().IsMulticast())
+    {
+        SuccessOrExit(error = AddDelayedResponse(*message, aMessageInfo.GetPeerAddr(),
```

Comment:
I like the idea of constructing the message first then simply delaying it.  One thing I'm concerned is possible message reordering due to the delays that can cause Frame Counter checks to fail and drop messages.  However, I'm not sure how often message drops will occur due to this behavior.  Thoughts?